10.4/js/script.js

```diff
@@ -0,0 +1,141 @@
+// maps
+window.initMap = function () {
+  var map = new google.maps.Map(document.getElementById('map'), {
+    zoom: 11,
+    center: slides[0].coords
+  });
+  var markers = [];
+  var currentSlide;
+  // add markers
+  for (var i = 0; i < slides.length; i++) {
+    markers[i] = new google.maps.Marker({
+      position: slides[i].coords,
+      map: map
+    });
+  }
+
+  markers[0].addListener('click', function(){
+    flkty.select(0);
+  });
```

Comment:
te listenery dodaj w petli for, ktora masz wyzej. 
Zeby to dzialalo musisz zrobic maly myk, zeby zasieg zmiennej `i` nie byl globalny. Najprosciej jak zamienisz 
`var i = 0` na `let i = 0`
Troche objasnien tutaj (zobacz ostatni komentarz): https://stackoverflow.com/questions/17750163/why-does-the-alert-coming-from-my-loop-always-return-the-last-value-not-each-it